src/pull_request.ts

```diff
@@ -53,19 +47,76 @@ export async function handlePullRequest() {
   let overviewComment = existingComments.find((comment) =>
     comment.body?.includes(OVERVIEW_MESSAGE_SIGNATURE)
   );
+  const isIncrementalReview = !!overviewComment;
+
+  // Maybe fetch review comments
+  const reviewComments = isIncrementalReview
+    ? (
+        await octokit.rest.pulls.listReviewComments({
+          ...context.repo,
+          pull_number: pull_request.number,
+        })
+      ).data
+    : [];
```

Comment:
The API request for review comments should be wrapped in a try-catch block to handle potential failures gracefully. Consider adding error handling:
```typescript
let reviewComments = [];
if (isIncrementalReview) {
  try {
    reviewComments = (await octokit.rest.pulls.listReviewComments({
      ...context.repo,
      pull_number: pull_request.number,
    })).data;
  } catch (error) {
    warning(`Failed to fetch review comments: ${error}`);
  }
}
```